articles/hosted-pages/login/lock.md

```diff
@@ -0,0 +1,25 @@
+---
+description: How to use Lock with the Hosted Login Page
+---
+# How to Use Lock with the Hosted Login Page
+
+[Lock](/libraries/lock) is a signup and authentication widget that provides quick and easy authentication for your application's users without requiring you to design the UI or interact manually with an Auth0 API.
+
+## Customize Lock in the Hosted Login Page
+
+The default login page for your tenant is a template that will use Lock to provide your users with an attractive interface and smooth authentication process, as discussed above. You can look over that template and use it as a starting point if you choose to customize it in any way.
+
+If you want to change any of Lock's [configurable options](/libraries/lock/v10/customization), you can do so using the [Hosted Pages](${manage_url}/#/login_page) editor interface. These options can alter the behavior of Lock itself, or the look and feel of the widget using the theming options. See the configuration documentation linked above for more details about how to configure Lock.
```

Comment:
I would change 

`See the configuration documentation linked above for more details about how to configure Lock.` 

to 

`See the [configuration documentation](/libraries/lock/v10/customization) for details on how to customize Lock.`